### ISSUE TITLE
Fix up OL7 security_patches_up_to_date before release

### ISSUE THREAD

**Issue by mpreisler** [OpenSCAP/scap-security-guide#2675]
Make sure:
```
-<Rule id="security_patches_up_to_date" severity="high" prodtype="ol7">
-<title>Ensure Software Patches Installed</title>
-<description>If the system is joined to the ULN or a yum server, run the following command to install updates:
-<pre>$ sudo yum update</pre>
-If the system is not configured to use one of these sources, updates (in the form of RPM packages)
-can be manually downloaded from the ULN and installed using <tt>rpm</tt>.
-<br /><br />
-NOTE: U.S. Defense systems are required to be patched within 30 days or sooner as local policy
-dictates.
-</description>
-<!-- Directly define remotely located OVAL to be executed when scanning this rule -->
-<check system="http://oval.mitre.org/XMLSchema/oval-definitions-5">
-  <check-content-ref href="https://linux.oracle.com/security/oval/com.oracle.elsa-all.xml.bz2" />
-</check>
-<ocil clause="updates are not installed">
-If the system is joined to the ULN or a yum server which provides updates, invoking the following command will
-indicate if updates are available:
-<pre>$ sudo yum check-update</pre>
-<br /><br />
-If the system is not configured to update from one of these sources,
-run the following command to list when each package was last updated:
-<pre>$ rpm -qa -last</pre>
-<br /><br />
-Compare this to Oracle Linux Security Advisories (ELSA) listed at
-<weblink-macro link="https://linux.oracle.com/pls/apex/f?p=105:21:::NO:RP::"/>
-to determine if the system is missing applicable updates.
-</ocil>
-<rationale>
-Installing software updates is a fundamental mitigation against
-the exploitation of publicly-known vulnerabilities. If the most
-recent security patches and updates are not installed, unauthorized
-users may take advantage of weaknesses in the unpatched software. The
-lack of prompt attention to patching could result in a system compromise.
-</rationale>
-<!-- Particular OVAL check is defined directly via <check> element above -->
-<!-- DO NOT REFERENCE an OVAL id here !!! -->
-<ref nist="SI-2,SI-2(c),MA-1(b)" disa="366" pcidss="Req-6.2" cis="1.8" srg="SRG-OS-000480-GPOS-00227" cjis="5.10.4.1" />
-</Rule>
```

is included in the OL7 benchmark before the next upstream release. We had to remove it because the new yaml system doesn't support this yet. We will be able to fix it once we start using jinja2 for templating.

**Comment by iokomin** [comment]
@mpreisler Is it possible to implement shared rule for rhel7 and ol7? 
In fact same yum based approach should be applied to check and fix system. 
It is clear description and ocil content could be modified to mention OS specific details like package providers. The question is about check itself which includes direct reference on OVAL definitions bundle. Is there any chance or sense to make it conditional depending on product? Which way for customization things like these would be recommended?

BTW with switch to yaml format direct OVAL check appeared to be missed in rhel7 'security_patches_up_to_date' rule. Thus it's a bit unclear what is the correct approach for implementing similar check for ol7:
```
<check system="http://oval.mitre.org/XMLSchema/oval-definitions-5">
  <check-content-ref href="https://linux.oracle.com/security/oval/com.oracle.elsa-all.xml.bz2" />
</check>
```

**Comment by mpreisler** [comment]
> @mpreisler was this fixed with the jinja merge?

yes, this has been fixed

**Comment by mpreisler** [comment]
Fixed by #2783
